### DISCUSSION TITLE
Prometheus remote write to cortex returns 405

### DISCUSSION THREAD

**Discussion by santoshpatil-pix** [cortexproject/cortex#4191] [Q&A] [answered]
I have just recently adopted cortex so apologies if the question comes across as  a very basic one.

I have deployed the cortex helm chart (https://github.com/cortexproject/cortex-helm-chart) to our EKS cluster with some changes to the values file.

To summarise,

- `auth_enabled` is  `false`
- In `nginx` I am setting some header values
` setHeaders:
      X-Scope-OrgID: 0
      X-Prometheus-Remote-Write-Version: 0.1.0`
- At the prometheus end I am setting up the `remoteWrite` url as follows
`remoteWrite:
      - url: http://cortex.dev.XXXXX.com/api/v1/push`
- Cortex nginx forwards the request to the cortex distributor which returns a 405

Am I missing something here? I would be really grateful If someone would provide some pointers on how to proceed from here.

Thanks.

**Comment by santoshpatil-pix** [comment, marked as answer]
I figured out that the issue was caused by using `http` URL instead of `https`. As a result nginx was converting the POST request to a GET request and was hence returning 405. Using `https` in the remote write URL fixed it for me. Apologies for the false alarm

**Reply by luchithapasqual** [reply]
Hi, Can ik how to check if the Cortex nginx forwards the request to the cortex distributor?
